packages/bpk-component-checkbox/README.md

```diff
@@ -26,15 +26,20 @@ export default () => (
 
 ## Props
 
-| Property   | PropType | Required | Default Value |
-| ---------- | -------- | -------- | ------------- |
-| name       | string   | true     | -             |
-| label      | node     | true     | -             |
-| required   | bool     | false    | false         |
-| disabled   | bool     | false    | false         |
-| white      | bool     | false    | false         |
-| smallLabel | bool     | false    | false         |
-| valid      | bool     | false    | null          |
+| Property      | PropType | Required | Default Value |
+| ------------- | -------- | -------- | ------------- |
+| name          | string   | true     | -             |
+| label         | node     | true     | -             |
+| required      | bool     | false    | false         |
+| disabled      | bool     | false    | false         |
+| white         | bool     | false    | false         |
+| smallLabel    | bool     | false    | false         |
+| valid         | bool     | false    | null          |
+| indeterminate | bool     | false    | false         |
```

Comment:
alphabetical

Reply:
They weren't before so wasn't sure - but can update it all

Reply:
Done